test/visual-diff/visual-tests

```diff
@@ -880,8 +880,8 @@
       "loading_complete_selectors": [".i-amphtml-story-360-loaded"]
     },
     {
-      "url": "examples/visual-tests/amp-story/amp-story-inline-page-attachment-dark-theme.html",
-      "name": "amp-story-inline-page-attachment-dark-theme: displays new inline CTA button with dark theme",
+      "url": "examples/visual-tests/amp-story/amp-story-page-attachment-v2.html",
+      "name": "amp-story-page-attachment-v2: displays new inline & outlink CTA buttons with various customizations",
```

Comment:
Nice, but each test is for one snapshot. 
We need a snapshot for each page in the template so Percy can diff them. 

I think we can do that with adding `#page=<page-id>`  to each test.
So, if we wanted to test the first page the `url` would be `examples/visual-tests/amp-story/amp-story-page-attachment-v2.html#page=cover`

In the next test, the `url` for page 2 would be `examples/visual-tests/amp-story/amp-story-page-attachment-v2.html#page=inline-custom-text`

We will know it's working when we see new snapshots show up for review in the PR.

We also need to be sure the experiment is on. 
It looks like that can be done right in the template by following the instructions in `visual-tests`:
https://github.com/ampproject/amphtml/blob/main/test/visual-diff/visual-tests#L75-L80

Reply:
I turned on the experiment! And I added a test for each page